specs/insert.test.ts

```diff
@@ -18,10 +22,57 @@ describe('insert', () => {
       'd',
       'e',
     ])
+
+    eq(insert(3, ['a', 'b'], list), [
+      'a',
+      'b',
+      'c',
+      ['a', 'b'],
+      'd',
+      'e',
+    ])
+
+    eq(insert(4, ['s', 't'], list), [
+      'a',
+      'b',
+      'c',
+      'd',
+      ['s', 't'],
+      'e',
+    ])
+
+    eq(insert(5, ['s', 't'], list), [
+      'a',
+      'b',
+      'c',
+      'd',
+      'e',
+      ['s', 't'],
+    ])
   })
 
   it('should append to the end of the list if the index is too large', () => {
     const list = ['a', 'b', 'c', 'd', 'e']
     eq(insert(8, 'z', list), ['a', 'b', 'c', 'd', 'e', 'z'])
   })
+
+  it('should append to the start of the list', () => {
+    const list = ['a', 'b', 'c', 'd', 'e']
+    eq(insert(0, 'z', list), ['z', 'a', 'b', 'c', 'd', 'e'])
+  })
+
+  it('should append element from the last of the list', () => {
+    const list = ['a', 'b', 'c', 'd', 'e']
+    eq(insert(-1, 'z', list), ['a', 'b', 'c', 'd', 'e', 'z'])
+  })
+
+  it('should test curried versions too', () => {
```

Comment:
These tests don't cover all the cases for curried versions. 

Check [this](https://github.com/Jozty/Fae/blob/master/specs/propIs.test.ts#L17) for all possible cases. Copy from this file, find and replace names, and arguments on from the copied code accordingly.

Also, add these tests for all 3-arry functions that you have worked upon with separate commit.

Reply:
Still, not all cases are covered
e.g
* `insert(0, _, _)('z')(list)`
and many more... 

Please do as stated in the previous comment